website/docs/policy/configuration.mdx

```diff
@@ -1,107 +0,0 @@
----
```

Comment:
deleted this file because users configure the agent through the profile so the content of the page mostly overlaps with the profile page. I move some bits from here to the profiles as well.